solara/components/datatable.py

```diff
@@ -19,7 +19,17 @@ def format_default(df, column, row_index, value):
 
 
 @solara.component
-def DataTable(df, page=0, items_per_page=20, format=None, column_actions: List[ColumnAction] = [], cell_actions: List[CellAction] = [], scrollable=False):
+def DataTable(
+    df,
+    page=0,
+    items_per_page=20,
+    format=None,
+    column_actions: List[ColumnAction] = [],
+    cell_actions: List[CellAction] = [],
+    scrollable=False,
+    on_column_menu_open: Optional[Callable[[Optional[str]], None]] = None,
+    column_widget=None,
```

Comment:
```suggestion
    column_menu_element : Optional[reacton.core.Element] = None,
```
If used from a component, it should be a element, in the widget layer it will indeed be translated to a widget. And for symmetry with `on_column_menu_open`, we can call this the column_menu_element I think.